mitodl/mitxpro
# #918 CourseRun Expiration Date

#### Pre-Flight checklist


#### What are the relevant tickets?
#918 

#### What's this PR do?
Adds `expiration_date` to the `CourseRun` model. The `CourseRunAdmin` is also updated to reflect the same. Serializers are also updated with the changes. There are validations in place for the `expiration_date` to be later than the `start_date` and/or `end_date` if any of them exist. Tests are included as well.

#### How should this be manually tested?
Go to django admin and pull up the add/change form for `CourseRun`. You should be able to interact with the `expiration_date` field as expected.


![Screenshot from 2019-08-09 15-06-35](https://user-images.githubusercontent.com/45350418/62771739-5935f280-bab7-11e9-9ff7-8203a7a4f7a1.png)
![Screenshot from 2019-08-08 18-52-26](https://user-images.githubusercontent.com/45350418/62771740-5935f280-bab7-11e9-9961-0639eaad28d6.png)
